op-service/sources/types_test.go

```diff
@@ -67,6 +68,7 @@ func TestBlockJSON(t *testing.T) {
 			err := block.verify()
 			if metadata.Fail {
 				require.NotNil(t, err, "expecting verification error")
+				require.ErrorContains(t, err, metadata.Reason, "validation failed for incorrect reason")
```

Comment:
I think you meant to update `TestBlockHeaderJSON` instead since we no longer generate block fixtures.

Reply:
We still have the block test fixtures, just they're manually tweaked to actually test what they're supposed to though.  So the script doesn't regenerate them or it would overwrite those tweaks.

Reply:
This is still the right test to update - it's still reading the block JSON data and applying it - it's just manually created test data now rather than generated by the script.

Reply:
gotcha. I'd prefer it if we can still generate the block + tweak fixtures. But this is fine too.

Reply:
Yeah I think the next step would be to have the script fetch valid data, then do the modifications to that in go - similar to the receipt validation unit tests I just added.  But I'm inclined to just ignore that until we next need to update these tests - likely as part of testing a new hard fork and then we can see if that's actually easier or not. I'm not sure there's really any reason to regenerate the existing data since it should always be identical.